src/components/Calculator.js

```diff
@@ -1,63 +1,54 @@
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-zero">0</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-dot">.</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-accent btn-equal">=</button>
         <button type="button" onClick={this.btnOnClick} className="btn-light btn-clear">AC</button>
         <button type="button" onClick={this.btnOnClick} className="btn-light btn-toogle-sign">+/-</button>
         <button type="button" onClick={this.btnOnClick} className="btn-light btn-percentage">%</button>
         <button type="button" onClick={this.btnOnClick} className="btn-accent btn-divider">/</button>
-        <button type="button" onClick={this.btnOnClick} className="btn-accent btn-multiply">*</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-accent btn-multiply">x</button>
         <button type="button" onClick={this.btnOnClick} className="btn-accent btn-substract">-</button>
         <button type="button" onClick={this.btnOnClick} className="btn-accent btn-plus">+</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-one">1</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-two">2</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-three">3</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-four">4</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-five">5</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-six">6</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-seven">7</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-eight">8</button>
-        <button type="button" onClick={this.btnOnClickOperator} className="btn-light btn-nine">9</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-one">1</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-two">2</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-three">3</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-four">4</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-five">5</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-six">6</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-seven">7</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-eight">8</button>
+        <button type="button" onClick={this.btnOnClick} className="btn-light btn-nine">9</button>
```

Comment:
- [ ] Nice job implementing the calculator buttons!  Now we need to meet this requirement:

![image](https://user-images.githubusercontent.com/57421823/152885644-0f14426f-80a4-480b-a4c2-69f382d14268.png)

Therefore, please kindly implement passing props to a child component, using lifecycle methods, and lifting the state, as explained in the lessons.

This requirement is meant to prevent you from struggling with later projects.  You can still keep it simple.

Reply:
Hi @Meltrust,

Do you mean I have to create a `component` type `button` to pass this state?

Reply:
Sorry for the late reply @hendridg.  I meant exactly that (among other things), and you did it beautifully.  

Congratulations on getting your project approved! 💪